toml.md

```diff
@@ -378,7 +378,8 @@ version of literal strings that solves this problem.
 **Multi-line literal strings** are surrounded by three single quotes on each
 side and allow newlines. Like literal strings, there is no escaping whatsoever.
 A newline immediately following the opening delimiter will be trimmed. All other
-content between the delimiters is interpreted as-is without modification.
+whitespace and newline characters remain intact. TOML parsers should feel free
+to normalize newline to whatever makes sense for their platform.
```

Comment:
How about the following?

```
**Multi-line literal strings** are surrounded by three single quotes on each
side and allow newlines. Like literal strings, there is no escaping whatsoever.
A newline immediately following the opening delimiter will be trimmed. Newline
normalisation is handled in the same manner as multi-line basic strings.

All other content between the delimiters is interpreted as-is without modification.
```

Reply:
Seems also good to me, however isn't the word "must" too strong here? Should be fine if a parser doesn't normalize?

Some context why I wrote the text the way I did:
- I reused sentences from multi-line basic string description to avoid any chance of discrepancy.
- I left out the "as-is without modification" sentence because it should be implied by "Like literal strings, there is no escaping whatsoever".

Reply:
> Should be fine if a parser doesn't normalize?

Yes.

If they don't normalise in multiline basic strings, then they must not normalise multiline literal strings either. And if they do normalise, they must normalise in both.

Doing it in one but not the other -- I can't think of any good reason to do that.

Reply:
Oh I see.

> Doing it in one but not the other -- I can't think of any good reason to do that.

Totally agreed. However, I'm not sure this is something that the spec needs to enforce. It's a small gotcha that an implementation could in theory get wrong. Since normalization is optional to begin with, it is still completely possible that the parsed content has mixed LFs and CRLFs (soon also CRs), so enforcing doesn't give much extra guarantees of the parsed output. Just sharing my thoughts, don't feel strongly at all about this :smile: 

The word "other" in the last sentence ("All other content...") used to reference the word "newline" ("A newline immediately..."). Now it's in another paragraph and there is an extra sentence in between so the meaning of "other" seems a lot more vague. Would it make sense to replace "other content" with "non-newline content" or similar expression? Assuming we want to keep the sentence, as I mentioned above, I feel it should already be implied.

Reply:
>Would it make sense to replace "other content" with "non-newline content" or similar expression?

The language suggested by @pradyunsg as stated is good. Its use of "other content" is clear in the context of the previous paragraph.